$MirOS: ports/devel/pango/patches/patch-configure_in,v 1.3 2007/01/06 16:30:17 bsiegert Exp $
--- configure.in.orig	Thu Jan 18 05:30:34 2007
+++ configure.in	Tue Jun 26 17:57:33 2007
@@ -206,7 +206,7 @@ if test "x$pango_os_win32" != xyes; then
     AC_MSG_WARN([X development libraries not found])
     have_x=false
   else
-    X_LIBS="$X_LIBS -lX11"
+    X_LIBS="$X_LIBS -lX11 -lXext"
     have_x=true
     AC_DEFINE(HAVE_X, 1, [Have X libraries])
   fi
@@ -239,7 +239,7 @@ if $have_fontconfig ; then
     LDFLAGS=$pango_save_ldflags
   fi
 
-  FREETYPE_LIBS="$FONTCONFIG_LIBS $FREETYPE_LIBS"
+  FREETYPE_LIBS="$FONTCONFIG_LIBS -lexpat $FREETYPE_LIBS"
   FREETYPE_CFLAGS="$FONTCONFIG_CFLAGS $FREETYPE_CFLAGS"
 
   AC_SUBST(FREETYPE_LIBS)
@@ -382,6 +382,8 @@ if $have_cairo ; then
     AC_DEFINE(HAVE_CAIRO_ATSUI, 1, [Whether Cairo uses ATSUI for fonts])
     have_cairo=true
   fi
+
+  XFT_LIBS="$XFT_LIBS -lXext"
 
   LDFLAGS=$pango_save_ldflags
 fi
